src/nvim/mbyte.c

```diff
@@ -1054,6 +1369,31 @@ bool utf_printable(int c)
 #endif
 }
 
+// Get unicode block of a Unicode character
+
+UnicodeBlocks utf_block(const int c)
+{
+  int bottom = 0;
+  int top = ARRAY_SIZE(unicode_block_table) - 1;
+  int middle = 0; //Start at zero so that latin chars are checked quickly
```

Comment:
hmm I think it could be clearer to spell this out as an extra `if()` statement in the beginning of the function (which how it typically is done).